spec/system/header/site_header_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
```

Comment:
Any particular reason these are split up into two different spec files (except that they were separate in the old qunit tests)? Kinda feels like they should be together?